source/wp-content/themes/wporg-showcase-2022/src/site-breadcrumbs/index.php

```diff
@@ -41,25 +41,23 @@ function render( $attributes, $content, $block ) {
 	}
 
 	global $pagename;
-	$post_ID = $block->context['postId'];
-	$post = get_post( $post_ID );
+	$title = ucfirst( strtolower( get_the_title() ) );
 
-	$title = $post->post_title;
-	if ( $pagename ) {
-		$title = ucfirst( strtolower( $pagename ) );
+	if ( 'archives' === $pagename ) {
```

Comment:
`/archives` would fail `is_page()` or `is_archive()` as it's considered as a `post`.
Originally I was thinking at least it should be a page, perhaps it no longer is when we specify it as a `Posts page`?
![image](https://user-images.githubusercontent.com/18050944/202918166-d1915877-4e84-4202-8354-bd77e857e158.png)

Reply:
The function to use for the "Posts page" is [`is_home`](https://developer.wordpress.org/reference/functions/is_home/), while the "Homepage" is [`is_front_page`](https://developer.wordpress.org/reference/functions/is_front_page/). And when the homepage displays latest posts, both are true on `/`.

Reply:
Added this update in https://github.com/WordPress/wporg-showcase-2022/commit/22b8d0b22576c0a0eaf295d4060824aae3fa065c.